src/js/components/SelectMultiple/SelectMultiple.js

```diff
@@ -0,0 +1,529 @@
+import React, {
+  forwardRef,
+  isValidElement,
+  useCallback,
+  useContext,
+  useMemo,
+  useState,
+  useRef,
+  useEffect,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { controlBorderStyle, useKeyboard, useForwardedRef } from '../../utils';
+import { defaultProps } from '../../default-props';
+
+import { Box } from '../Box';
+import { DropButton } from '../DropButton';
+import { Keyboard } from '../Keyboard';
+import { FormContext } from '../Form/FormContext';
+import { SelectMultipleValue } from './SelectMultipleValue';
+
+import { SelectMultipleContainer } from './SelectMultipleContainer';
+import {
+  HiddenInput,
+  SelectTextInput,
+  StyledSelectDropButton,
+} from '../Select/StyledSelect';
+import {
+  applyKey,
+  getNormalizedValue,
+  changeEvent,
+  getSelectIcon,
+  getIconColor,
+  getDisplayLabelKey,
+} from '../Select/utils';
+import { DefaultSelectTextInput } from '../Select/DefaultSelectTextInput';
+import { MessageContext } from '../../contexts/MessageContext';
+import { SelectMultiplePropTypes } from './propTypes';
+
+const StyledSelectBox = styled(Box)`
+  ${(props) => !props.callerPlain && controlBorderStyle};
+  ${(props) =>
+    props.theme.select &&
```

Comment:
With the way `?.` works, do we need the intervening checks still? Can this just be `${(props) => props.theme.select?.control?.extend}`

Reply:
Changed to `${(props) => props.theme.select?.control?.extend}` in latest commit